ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/text/v1/text/clientlibs/site/js/textview.js

```diff
@@ -61,6 +61,14 @@
             setFocus() {
                 this.setActive();
             }
+
+            updateValue(value) {
+                // html sets undefined value as undefined string in input value, hence this check is added
+                let actualValue = typeof value === "undefined" ? null :  value;
```

Comment:
in what scenario will we get undefined as value from model ?
Also if the value is undefined? is putting null recommended or should we put an empty string?will it cause any difference in behaviour?